### PR TITLE
Implement Material-UI and new folder structure for AddItems.js

### PR DESCRIPTION
**Pull request by dariatsvetkova**
_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._ 

## Description

1. Implement new folder structure;
2. Use Material UI components for `AddItems.js`:
* TextField
* FormControl
* FormLabel
* FormControlLabel
* RadioGroup
* Radio
* Button
***Proper styles coming soon***

## Related Issue

#14 

## Acceptance Criteria

- As a user, I want the app to look professional and welcoming

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|  ✓  | :hammer: Refactoring       |

### Before

screencapture-tcl-22-shopping-list-pr47-cf-material-ui-welco-1fukodsg-web-app-additems-2021-05-19-18_24_30

### After

screencapture-localhost-3000-additems-2021-05-19-20_55_04

## Testing Steps / QA Criteria

To test the feature, use one of the following ways:

1. Follow the [preview link](https://tcl-22-shopping-list--pr50-dt-mui-add-items-ddp8uu0t.web.app)
2. Launch the app locally from your terminal:
- Pull down this branch with `git pull origin dt-mui-add-items` and check that branch out with `git checkout origin dt-mui-add-items`
- Run npm start to launch the app in your browser at http://localhost:3000

In the app, navigate to Add Groceries view to see the changes.

### CODE REVIEW

File: src/pages/AddItems/AddItems.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+  const radioBtnHandler = (event) => {
+    setItemFreq(+event.target.value);
+  };
+
+  return (
+    <div>
+      <StyledForm id="add-item-form" onSubmit={submitGroceryItem}>
+        <TextField
+          id="item"
+          label="Item name:"
```
Comment (dariatsvetkova) [inline src/pages/AddItems/AddItems.js:60]: @ablizben this is the subtle label for the text field that I was talking about

**Review by laurenmbeatty — APPROVED** [review]
Great! I like the new organization. The header font looks slick too. I don't see any functionality regressions. 👍

**Review by cafloyd — APPROVED** [review]
Looks awesome!

File: src/pages/AddItems/AddItems.js (hunk 1 above)
Comment (ablizben) [reply src/pages/AddItems/AddItems.js:60]: Thank you!

**Review by ablizben — APPROVED** [review]

**Comment by kkatoch89** [comment]
Looks great!

**Pull request merged**